File: files/addedRecord.cpp
#include<iostream>
#include<fstream>
using namespace std;
int main(){
	ofstream file("C:\\Users\\Seyrek\\Desktop\\C++\\files\\text.txt", ios :: app);
	file << "\nYalniz senin gozlerini";
	file.close();
	return 0;
}

File: files/addedRecord.java
package springIntro;

import java.io.*;

public class addedRecord {

	public static void main(String[] args) throws IOException{
		File file = new File("C:\\Users\\Seyrek\\eclipse-workspace\\springIntro\\src\\springIntro\\text.txt");
		BufferedWriter print = new BufferedWriter(new FileWriter(file, true));
		String str;
		str = "Yalniz seni, yalniz senin gozlerini";
		print.write(str);
		print.close();
	}

}
